api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -77,51 +80,69 @@ message GetAccessListRequest {
-message GetAccessListMembersRequest {
-  // name is the name of the access list to retrieve.
+// UpsertAccessListRequest is the request for upserting an access list.
+message UpsertAccessListRequest {
+  // access_list is the access list to upsert.
+  AccessList access_list = 1;
+}
+
+// DeleteAccessListRequest is the request for deleting an access list.
+message DeleteAccessListRequest {
+  // name is the name of the access list to delete.
   string name = 1;
 }
 
-// GetAccessListMembersResponse is the response for retrieving members of an access list.
-message GetAccessListMembersResponse {
-  // members are the members of the acccess list.
+// DeleteAllAccessListsRequest is the request for deleting all access lists.
+message DeleteAllAccessListsRequest {}
+
+// ListAccessListMembersRequest is the request for getting paginated access list members.
+message ListAccessListMembersRequest {
+  // page_size is the size of the page to request.
+  int32 page_size = 1;
+
+  // page_token is the page token.
+  string page_token = 2;
+}
+
+// ListAccessListMembersResponse is the response for getting paginated access list members.
+message ListAccessListMembersResponse {
+  // members is the list of access list members.
   repeated Member members = 1;
+  // next_page_token is the next page token.
+  string next_page_token = 2;
 }
 
-// UpsertAccessListRequest is the request for upserting an access list.
-message UpsertAccessListRequest {
-  // access_list is the access list to upsert.
-  AccessList access_list = 1;
+// GetAccessListMemberRequest is the request for retrieving an access list member.
+message GetAccessListMemberRequest {
+  // access_list is the name of the access list that the member belongs to.
+  string access_list = 1;
+
+  // member_name is the name of the user that belongs to the access list.
+  string member_name = 2;
 }
 
 // UpsertAccessListMemberRequest is the request for upserting an access list member.
 message UpsertAccessListMemberRequest {
-  // access_list is the name of access list.
-  string access_list = 1;
-  // name is the name of the user to add.
-  string name = 2;
-  // reason is the reason the user is being added.
-  string reason = 3;
-}
+  // Renamed the "name" field to "member_name".
+  reserved 1, 2, 3;
+  reserved "access_list", "name", "reason";
 
-// DeleteAccessListRequest is the request for deleting an access list.
-message DeleteAccessListRequest {
-  // name is the name of the access list to delete.
-  string name = 1;
+  // member is the access list member to upsert.
+  Member member = 4;
 }
 
 // DeleteAccessListMemberRequest is the request for deleting a member from an access list.
 message DeleteAccessListMemberRequest {
+  // Renamed the "name" field to "member_name".
+  reserved 2;
+  reserved "name";
+
   // access_list is the name of access list.
   string access_list = 1;
-  // name is the name of the user to delete.
-  string name = 2;
```

Comment:
Ditto for this change, any impact on existint clients?

Reply:
New as well.